backend/src/Entity/ResetPasswordRequest.php

```diff
@@ -35,15 +35,12 @@ class ResetPasswordRequest implements ResetPasswordRequestInterface
      */
     private ?UuidInterface $id = null;
 
-    /**
+    public function __construct(/**
      * @ORM\ManyToOne(targetEntity="User")
      * @ORM\JoinColumn(nullable=false)
      */
-    private User $user;
-
-    public function __construct(User $user, \DateTimeInterface $expiresAt, string $selector, string $hashedToken)
+    private User $user, \DateTimeInterface $expiresAt, string $selector, string $hashedToken)
```

Comment:
this constructor looks really ugly with this indentation, maybe it can be better

Reply:
How is the right/pretty way of fixing this mess? 
```
    public function __construct(
        /**
         * @ORM\ManyToOne(targetEntity="User")
         * @ORM\JoinColumn(nullable=false)
         */
        private User $user,
```

Reply:
I think it looks better now. Keep in mind normally those properties won't get moved to the constructor, this is a special case, I don't mind if it looks like this (if it works, obv)